examples/spec/integration/child_workflow_terminated_workflow_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'workflows/child_workflow_terminated_workflow.rb'
+
+describe ChildWorkflowTerminatedWorkflow do
+  subject { described_class }
+
+  it 'successfully can catch if a child workflow times out' do
+    workflow_id = SecureRandom.uuid
+
+    Temporal.start_workflow(
+      subject,
+      options: { workflow_id: workflow_id }
+    )
+
+    result = Temporal.await_workflow_result(
+      subject,
+      workflow_id: workflow_id
+    )
+
+    puts result
```

Comment:
Looks like this is a leftover from debugging the spec. It also seems to be missing an expectation

Reply:
I guess it's supposed to be `expect(result).to eq(child_workflow_terminated: true)`

Reply:
agreed! added this to both places